src/vsphere_cpi/lib/cloud/vsphere/cloud.rb

```diff
@@ -767,5 +811,16 @@ def disk_configurations(vm_type, existing_disk_cids=[])
       )
       disk_configurations.push(ephemeral_disk_config)
     end
+
+    #This subnet will be used in create_logical_router_port.
+    # ip_addresses is going to be a gateway IP for subnet - aka IP for router.
+    def create_subnet_obj(range, gateway)
+      if (!range.nil? && range.include?('/'))
```

Comment:
Use a proper regular expression here. This won't be caught by this logic:

```
192.168.1.1/24/24
```

This probably works: https://www.regextester.com/93987

Reply:
probably should use netaddr gem (part of standard library) to parse the range.

Reply:
@cppforlife On my end it looks like netaddr isn’t part of the standard library which is why I recommended the Regexp. Did it get added recently?

Reply:
Going with `netaddr`